angr/analyses/cfg_base.py

```diff
@@ -1226,7 +1226,11 @@ def _process_irrational_functions(self, functions, predetermined_function_addrs,
 
             if not has_unresolved_jumps:
                 continue
-
+            
+            if function.startpoint == None:
```

Comment:
Oh forgot to say that I'd rather use `if function.startpoint is None` instead of `==`.